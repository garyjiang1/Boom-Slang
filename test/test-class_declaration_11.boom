class MyObjectTwo:
	required:
		string x


	optional:
		string z = "foo"
MyObjectTwo ot = MyObjectTwo("two")
println(ot.z)
println(ot.x)
MyObjectTwo ot = MyObjectTwo("two","foo2")
println(ot.z)
println(ot.x)
