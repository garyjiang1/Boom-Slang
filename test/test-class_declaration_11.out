foo
two
foo2
two
